aws/kustomization.yaml

```diff
@@ -0,0 +1,26 @@
+commonAnnotations:
+  giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+
+resources:
```

Comment:
Can we somehow generate this? We have to duplicate the info otherwise and this is very easy to go wrong.

Reply:
Agreed. As I was moving things around I realized we could have a generator, maybe `devctl create release` which initializes the directory and updates the root list. It could base off of another release.

Reply:
Could be one idea to improve this :+1:

Reply:
For now I will create a checklist so we don't miss one of these manual steps.

Reply:
I actually turned the checks into unit tests so it will be impossible to make these mistakes.